scripts/run_multiple_configurations.py

```diff
@@ -194,6 +195,34 @@ def make_configurations(base_config, changing_configurations, settings):
 settings = raw_config["Multiple Configurations Settings"]
 changing_configurations = raw_config["Multiple Configurations Values"]
 
+# Detect the possible configuration combinations of the "Multiple Configurations
+# Settings" and store them in the right order. This information will be written
+# in the run_combination.yaml file as a comment, so that the rename_run_folders.sh
+# can easily parse the comments and rename the run folders automatically.
+base_model_dir = base_config['Logging']['model_directory']
+base_scripts_dir = base_config['Logging'].get('scripts_directory', '')
+multiple_config_values_all = raw_config["Multiple Configurations Values"]
+multiple_config_values = []
+for key in multiple_config_values_all.keys():
+    values = []
+    for val in multiple_config_values_all[key]['values'].keys():
+        values.append(val)
```

Comment:
This would not work as expected for range values (e.g. grid/random search over learning rate). I think you have to just number them using the `config['Multiple Configurations Values']['values']['num_values']` parameter (or `config['Multiple Configurations Settings']['num_grouped_range_values']` if num_values is null).